docs/narr/i18n.rst

```diff
@@ -270,12 +270,12 @@ system run this command:
    $ sudo apt-get install gettext
 
 Installing Lingua is done with the Python packaging tools. If the
-:term:`virtualenv` into which you've installed your :app:`Pyramid` application
-lives in ``/my/virtualenv``, you can install Lingua like so:
+:term:`virtual environment` into which you've installed your :app:`Pyramid`
+application lives in ``/my/venv``, you can install Lingua like so:
 
 .. code-block:: text
 
-   $ cd /my/virtualenv
+   $ cd /my/venv
    $ $VENV/bin/easy_install lingua
```

Comment:
Most of this section seems to have been written without regard to how the rest of the chapters handle virtualenvs. There is not really a need here to show the `cd /my/venv` and discuss `/my/venv` at all. I'd just show the example as `$ $VENV/bin/pip install lingua` and be done with it. Similar for windows down below.